startupscript/gcp/resource-mount.sh

```diff
@@ -11,21 +11,35 @@
 # - Workbench CLI is installed
  
 if ! which gcsfuse >/dev/null 2>&1; then
-  emit "Installing gcsfuse..."
+  if type apk > /dev/null 2>&1; then
+    emit "Installing gcsfuse from source..."
 
-  # install packages needed to install gcsfuse
-  apt-get install -y \
-    fuse \
-    lsb-release
+    apk add --no-cache fuse git go
 
-  # Install based on gcloud docs here https://cloud.google.com/storage/docs/gcsfuse-install.
-  GCSFUSE_REPO="gcsfuse-$(lsb_release -c -s)"
-  readonly GCSFUSE_REPO
```

Comment:
Can you add a reference to where these install steps come from? Is there a doc like was referenced in the existing comment.

Reply:
Oh its from the same page, but the comment is moved below into the "apt" section.  I can put it at the top or soemthing.

Reply:
updated comment